### PR TITLE
Add a method to clear a post out of the cache

### LINKED ISSUE #7: Expose a method to clear the cache for a given post
Required by https://github.com/ArtskydJ/noddity-render-dom/issues/24

The butler isn't in charge of parsing.  But if some post doesn't parse, consumers need to be able to signal to the butler that it is uncool, and should not be cached in the way that other posts are cached.

This may just require a `clearCache(postName)` method.  Reading the code and sleeping on it could confirm.

### PR DESCRIPTION
**Pull request by ArtskydJ**
Fixes #7.

### CODE REVIEW

**Comment by TehShrike** [comment]
lgtm

File: readme.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ Returns true if all of the posts listed in the index.json file have been downloa
 
 Updates the cache by downloading the latest version of the post from the server.  The optional callback returns the post.
 
+### butler.removePost(filename, [cb])
+
+Removes the post from the cache.
```
Comment (ArtskydJ) [inline readme.md]: The lack of docs is a little disappointing.

Comment (ArtskydJ) [inline readme.md]: (which can be fixed later...)

**Comment by TehShrike** [comment]
The tests didn't pass for me after a merge, even after running npm prune/npm install.  We need to depend on the updated version of levelup-cache

**Comment by TehShrike** [comment]
I pushed that up in https://github.com/TehShrike/noddity-butler/commit/188ea1514f9673b213430bf07eb96161d23e232d

**Comment by TehShrike** [comment]
Published as 2.6.0

**Comment by ArtskydJ** [comment]
Whoops, good call.

**Pull request merged**